README.md

```diff
@@ -35,6 +35,115 @@ Node.js is a prerequisite for installing tedious. Once you have installed [Node.
 - [Node.js + Ubuntu](https://www.microsoft.com/en-us/sql-server/developer-get-started/node/ubuntu/)
 - [Node.js + Windows](https://www.microsoft.com/en-us/sql-server/developer-get-started/node/windows/)
 
+### Connecting to a database
+```js
+  const Connection = require('tedious').Connection;
+
+  const config = {
+    server: "192.168.1.210", // or "localhost"
+    options: {},
+    authentication: {
+      type: "default",
+      options: {
+        userName: "test",
+        password: "test",
+      }
+    }
+  };
+
+  const connection = new Connection(config);
+
+  // Setup event handler when the connection is established.
+  connection.on('connect', function(err) {
+    if(err) {
+      console.log('Error: ', err)
+    }
+    // If no error, then good to go...
+    executeStatement();
+  });
+
+  // Initialize the connection.
+  connection.connect();
+```
+### Setting up a SQL Server in Docker
```

Comment:
These steps seem to be windows specific, but use the linux container. Does this work out of the box on windows? What about using the windows container? Should we also have getting started steps for that?